### PR TITLE
fix float32/64 decoding from edgex bug

### PR DESCRIPTION
**Pull request by worldmaomao**

### CODE REVIEW

File: xstream/extensions/edgex_source.go [hunk 1]
```diff
@@ -215,16 +215,15 @@ func (es *EdgexSource) getValue(r models.Reading, logger api.Logger) (interface{
 
 func (es *EdgexSource) getFloatValue(r models.Reading, logger api.Logger) (interface{}, error) {
 	if len(r.FloatEncoding) == 0 {
-		if strings.Contains(r.Value, "==") {
+		if strings.Contains(r.Value, "=") {
```
Comment (jinfahua) [inline xstream/extensions/edgex_source.go:218]: Not sure if there are some cases that only include one '='?  I used same logic as in [this link](https://github.com/lenny-intel/app-service-examples/blob/refactor/app-services/advanced-filter-convert-publish/functions/convert.go#L60).

Comment (worldmaomao) [reply xstream/extensions/edgex_source.go:218]: pleae refer this post:
https://www.cnblogs.com/antineutrino/p/3756106.html

Comment (jinfahua) [reply xstream/extensions/edgex_source.go:218]: ok, thanks for your info.

**Comment by jinfahua** [comment]
The code is OK, could you please re-create PR, and set target branch to `develop`. The `master` branch is for stabled release. Thank you!

**Comment by worldmaomao** [comment]
> The code is OK, could you please re-create PR, and set target branch to `develop`. The `master` branch is for stabled release. Thank you!

ok.I will close this PR and recreate new PR.

**Pull request closed without merging**